plugins/outputs/mqtt/sample.conf

```
@@ -2,9 +2,14 @@
 [[outputs.mqtt]]
   ## MQTT Brokers
   ## The list of brokers should only include the hostname or IP address and the
-  ## port to the broker. This should follow the format '{host}:{port}'. For
-  ## example, "localhost:1883" or "127.0.0.1:8883".
-  servers = ["localhost:1883"]
+  ## port to the broker. This should follow the format `[{scheme}://]{host}:{port}`. For
+  ## example, `localhost:1883` or `mqtt://localhost:1883`.
+  ## Scheme can be any of the following: tcp://, mqtt://, tls://, mqtts://
+  ## non-TLS and TLS servers can not be mix-and-matched.
+  servers = ["localhost:1883", ] # or ["mqtts://tls.example.com:1883"]
+
+  # Protocol can be `3.1.1` or `5`. Default is `3.1.1`
+  procotol = "3.1.1"
```

Please leave this commented to show it's optional. The value show should then be the default value...
```suggestion
  ## Protocol can be `3.1.1` or `5`. Default is `3.1.1`
  # procotol = "3.1.1"
```